<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEditYan">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserZ">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZ">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButtonZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2Z">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
